src/index.js

```diff
@@ -1,13 +1,58 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-      // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+const tasksArr = [
+  {
+    index: 0,
+    describtion: 'Wake up',
+    completed: false,
+  },
+  {
+    index: 1,
+    describtion: 'Go to shower',
+    completed: false,
+  },
+  {
+    index: 2,
+    describtion: 'Take breakfast',
+    completed: false,
+  },
+  {
+    index: 3,
+    describtion: 'start class',
+    completed: false,
+  },
+];
```

Comment:
- [ ] You have a grammar error here, is "description", and no "describtion", kindly fix this 😄